src/cli.rs

```diff
@@ -210,43 +210,49 @@ fn build_command() -> Command<'static> {
                 .takes_value(true)
                 .value_name("UNIT")
                 .possible_values(&["millisecond", "second"])
-                .help("Set the time unit to be used. Possible values: millisecond, second."),
+                .help("Set the time unit to be used (default: second). Possible values: millisecond, second.\
```

Comment:
It's not true that the default is "second". By default, the unit will be auto-determined from the mean time of the first benchmark (IIRC)